src/shared/components/History/index.tsx

```diff
@@ -45,10 +46,20 @@ const HistoryComponent: React.FunctionComponent<{
               <FriendlyTimeAgo date={moment(revision.updatedAt)} />
               <div
                 className="changes"
-                style={{ width: 'max-content', marginTop: '1em' }}
+                style={{ width: '100%', marginTop: '1em' }}
               >
                 {revision.hasChanges ? (
-                  <Card>{JSON.stringify(revision.changes, null, 2)}</Card>
+                  <Card>
+                    <pre style={{ width: '100%' }}>
+                      {JSON.stringify(
+                        revision.changes,
+                        (key, value) => {
+                          return typeof value === 'undefined' ? null : value;
```

Comment:
If it is the case that `value` can be null also then it may make sense to protect against that too (maybe using isNil from lodash). Otherwise this is good.

Reply:
here, we are checking if the key/value is exist in the object, basically the diff method, will assign the value undefined when it can not find it in the object, in the resource objects we will not find the null value, since `null` it's not available in python (mostly used here).